hippunfold/workflow/scripts/create_warps.py

```diff
@@ -100,23 +100,30 @@ def summary(name, array):
 # we have points defined by coord_flat_{ap,pd,io}, and corresponding value as native_coords_phys[:,i]
 # and we want to interpolate on a grid in the unfolded space
 
-# unnormalize and add a bit of noise so points don't ever perfectly overlap
-coord_flat_ap_unnorm = coord_flat_ap * unfold_dims[0]
-coord_flat_pd_unnorm = coord_flat_pd * unfold_dims[1]
-coord_flat_io_unnorm = coord_flat_io * unfold_dims[2]
-
-# get unfolded grid
-
-points = np.stack(
-    [
-        coord_flat_ap * unfold_dims[0]
-        + (np.random.rand(coord_flat_ap.shape[0]) - 0.5) * 1e-6,
-        coord_flat_pd * unfold_dims[1]
-        + (np.random.rand(coord_flat_ap.shape[0]) - 0.5) * 1e-6,
-        coord_flat_io * unfold_dims[2]
-        + (np.random.rand(coord_flat_ap.shape[0]) - 0.5) * 1e-6,
-    ],
-    axis=1,
+# add some noise to avoid perfectly overlapping datapoints!
+points = (
+    coord_flat_ap * unfold_dims[0]
+    + (np.random.rand(coord_flat_ap.shape[0]) - 0.5) * 1e-6,
+    coord_flat_pd * unfold_dims[1]
+    + (np.random.rand(coord_flat_ap.shape[0]) - 0.5) * 1e-6,
+    coord_flat_io * unfold_dims[2]
+    + (np.random.rand(coord_flat_ap.shape[0]) - 0.5) * 1e-6,
+)
+
+# get unfolded grid (from 0 to 1, not world coords), using meshgrid:
+#  note: indexing='ij' to swap the ordering of x and y
+epsilon = snakemake.params.epsilon
```

Comment:
I know this PR is merged already, but easier to comment here than anew -- these changes break the rule because epsilon is not a parameter in the rule. Also, unfold_gx/unfold_gy/unfold_gz get create here, but don't actually get used .. so it seems we can just revert the create_warps changes to fix the problem, unless you were trying to do something else here?

Reply:
oh yeah, good point. Looks like we were never actually using the epsilon that we'd previously defined? I'll make a small PR to patch this & 1 other small bug i found

Also yeah `points` and `unfold_gx` somehow became redundant. We can cut the latter.